app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +35,8 @@
         <activity
             android:name="com.parishod.watomatic.activity.customreplyeditor.CustomReplyEditorActivity"
             android:label="@string/mainAutoReplyLabel"
-            android:parentActivityName="com.parishod.watomatic.activity.main.MainActivity">
+            android:parentActivityName="com.parishod.watomatic.activity.main.MainActivity"
+            android:exported="true">
```

Comment:
What does this do?

Reply:
In latest version we need to explicitly mention if the activities will be Accessed from outside.